torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -190,14 +193,23 @@ def from_float(
         preserve_zero: bool = True,
         zero_point_domain: ZeroPointDomain = ZeroPointDomain.INT,
         layout_type: LayoutType = PlainLayoutType(),
+        use_hqq: bool = False,
```

Comment:
maybe fine for now, but we could work on generalizing affine quantized tensor subclass I feel

Reply:
Okay great, if you have some example I can follow I can do it

Reply:
one thing I can think of now is just to do `AffineQuantizedTensor.from_float_hqq` and use hqq quant primitive ops there

Reply:
Is this confirmed with @HDCharles or we keep the `use_hqq` flag?

Reply:
yeah, this is fine for now I think